flumine/markets/blotter.py

```diff
@@ -94,23 +95,50 @@ def process_cleared_orders(self, cleared_orders) -> list:
     """ position """
 
     def selection_exposure(self, strategy, lookup: tuple) -> float:
```

Comment:
This gets called a lot, any idea of how much slower this changes makes it?

Reply:
Running 1000 backtests on my personal code...
For a strategy that hits prices:
36.7 seconds with this PR.
36.4 seconds without.

For a strategy that works prices:
37.03 seconds with this PR
37.4s without

So the results are within variance of each other.

However, it's probably bad for you to just accept my figures. Let me put together an example script later so that you can check on your own machine.

Reply:
Actually, you can just test the performance using the existing integration test.

I don't think it makes much, if any, difference.

To simulate more activity I changed

                    if runner.last_price_traded < 2:

to 

                    if runner.last_price_traded <= 1000.0: